Tests/LibWeb/TestCSSPixels.cpp

```diff
@@ -87,4 +87,40 @@ TEST_CASE(saturated_subtraction)
     EXPECT_EQ(value - -1, CSSPixels(INFINITY));
 }
 
+TEST_CASE(rounding)
```

Comment:
tests for rounding is nice! maybe some tests for changes in `*` and `/` would be even better :)